exercises/rational-numbers/Example.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+
+public static class RationalNumbers
+{
+    private const int NUMERATOR_INDEX = 0;
```

Comment:
Could you use PascalCasing here? To better adhere to the default [naming conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members).